platform/darwin/resources/fr.lproj/Foundation.strings

```diff
@@ -295,3 +298,9 @@
 /* OpenStreetMap short name attribution */
 "OSM_SHORT_NAME" = "OSM";
 
+/* User-friendly error description */
+"REMOVE_SRC_FAIL_IN_USE_FMT" = "La source '%@' est en cours d'utilisation, impossible de la supprimer.";
+
+/* User-friendly error description */
+"REMOVE_SRC_FAIL_MISMATCH_FMT" = "L'identifiant '%1$@' ne correspond pas à l'identifiant de la source '%2$s'";
```

Comment:
I made some minor punctuation changes to three strings in this file on Transifex, but anyways #13492 will change the wording of these strings once we merge that PR.